CHANGELOG.md

```diff
@@ -19,6 +19,25 @@ scztt, simdax, smoge, smrg-lm, snappizz, telephon, thormagnusson,
 tiagomoraismorgado88, timsutton, vivid-synth, vividsnow, yurivict, and many
 more in the SC community who helped in ways other than participation on GitHub.
 
+Known Issues
+------------
+
```

Comment:
We should add: 
- File.copy crashes interpreter when destination exists ([#3401](https://github.com/supercollider/supercollider/issues/3401)).